<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pokedeck</class>
 <widget class="QDialog" name="pokedeck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="perso_avec_pokemon">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images_test/perso_avec_pokemon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bulbasaur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Squirtle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Charmander</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="fond_prairie">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-8</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../projet_pok√©mon/pokemon_lagourgue_manceau_pelissier/interface_graphique/images/prairie_et_ciel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>fond_prairie</zorder>
  <zorder>perso_avec_pokemon</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
